test/lev-report/__snapshots__/History.test.js.snap

```diff
@@ -0,0 +1,3816 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
It looks like we previously decided not to commit these files.

Do we need something added to `.gitignore`?

Reply:
All `.snap` files must be committed along with the code, otherwise the tests won't work (or they will present false failures at least)

Reply:
They don't seem to be in the repo at present. (Unless I was looking in the wrong place.)

Reply:
I can see the snapshots that have already been created here: https://github.com/UKHomeOffice/lev-react-components/tree/master/test/lev-report/charts/__snapshots__
This history one won't be in there until this PR is merged though.

Reply:
Okay.